gobblin-salesforce/src/main/java/org/apache/gobblin/salesforce/SalesforceConnector.java

```diff
@@ -83,6 +83,10 @@ public HttpEntity getAuthentication() throws RestApiConnectionException {
       String password = PasswordManager.getInstance(this.state)
           .readPassword(this.state.getProp(ConfigurationKeys.SOURCE_CONN_PASSWORD));
       String securityToken = this.state.getProp(ConfigurationKeys.SOURCE_CONN_SECURITY_TOKEN);
+      if (this.state.getPropAsBoolean(ConfigurationKeys.SOURCE_CONN_DECRYPT_SECURITY_TOKEN, false)) {
```

Comment:
Why have the option? The password manager will pass the password through if it is not encoded.

Reply:
Agreed it's not required. Fixed it.